library/windows/BUILD.gn

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build/packaging.gni")
+import("//library/engine.gni")
+import("//library/windows/glfw.gni")
+
+# Allows targets depending on this library to use library-style
+# inculdes for its header rather than project-relative.
+config("relative_public_headers") {
```

Comment:
Let's fold this into library/BUILD.gn since two platforms are using it, rather than duplicating it and having different paths to reference it by platform.